modules/gitops-repo-server-properties.adoc

```diff
@@ -9,20 +9,17 @@
 The following properties are available for configuring the Repo server component:
```

Comment:
Similarly to the table above, here are also some missing properties that might be considered to be added: `extraRepoCommandArgs`, `initContainers`, `remote`, `sidecarContainers`, `volumeMounts`, and `volumes`.

Reply:
Thanks @eromanova97 for the observation and request - Here is the JIRA: https://issues.redhat.com/browse/RHDEVDOCS-5940